hack/manifest-gen/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang as builder
```

It might be good to pin a version to avoid surprises.
```suggestion
FROM golang:1.14 as builder
```